tor/core/admin_commands.py

```diff
@@ -1,15 +1,87 @@
+import json
 import logging
 import random
 
 from praw.exceptions import ClientException as RedditClientException
 # noinspection PyProtectedMember
-from tor_core.helpers import _
 from tor_core.helpers import clean_id
 from tor_core.initialize import initialize
 
 from tor.core.user_interaction import process_done
 
 
+def process_command(reply, config):
+    """
+    This function processes any commands send to the bot via PM with a subject
+    that stars with a !. The basic flow is read CSV file, look for row with same
+    subject, check if it needs mod, and check for mod, and then reply to the
+    PM with the results of the function listed in the CSV file.
+
+    To add a new command: add an entry to commands.csv, (look at the top line
+    for row names) add your function to command_funcs, and add your function.
+    :param reply:
+    :param config:
+    :return:
+    """
```

Comment:
We're making a ton of references in the docstring to a CSV, yet we're using a JSON file. Looks like this could use revisiting to make sure it is accurate.

Reply:
blorp, that must have been from when I was using CSV